pkg/services/sqlstore/dashboard_version.go

```diff
@@ -67,49 +67,68 @@ func GetDashboardVersions(query *models.GetDashboardVersionsQuery) error {
 	return nil
 }
 
-const MAX_VERSIONS_TO_DELETE = 100
+const MAX_VERSIONS_TO_DELETE_PER_BATCH = 100
```

Comment:
Better. But we are still capped at deleting 30k dashboards versions per hour. This somehow makes me think of the Bill Gates quote "640K is more memory than anyone will ever need on a computer".

Reply:
So you suggests adding configuration for this?

Reply:
NO. No more configuration :)

I think this will solve our problem on HG and if someone else has problems with the 30k limit we can reopen the issue. Since we have this limit it's not 100% solved. But its good enough to not care about it for (hopefully) a long time.

Reply:
Yeah, I wanted to avoid the situation where on first run for a backlogged instance it would spend ~forever deleting potentially lots of old dashboard versions.  Realistically if someone is creating more than 30k dashboard versions per hour they're going to be running into other issues as well, so at risk of having to wear the dunce hat I think it should be high enough for now.